imds-filterd/ident.c

```diff
@@ -54,6 +54,7 @@ gotdata(void * cookie, ssize_t len)
 	struct cstate * cs = cookie;
 	struct sockaddr_in addrs[2];
 	struct xucred uc;
+	int alloclen;
```

Comment:
Why "alloclen"?  Seems like "printlen" would make more sense?

Reply:
No particular reason.  I mean, first I went `len`, then figured that wasn't specific enough, then I thought `strlen`, then I realized that it would be insane to copy a function name from the standard library.  I'll switch it to `printlen`.